### DISCUSSION TITLE
Dependency update challenge

### DISCUSSION THREAD

**Discussion by ghost** [webdriverio/webdriverio#11545] [General]
I have recently been updating from 8.6.* to 8.20.* across my WDIO dependencies and have an encountered an odd issue where I get the following error shown in the wdio-0-0.log. 

Error is:
`@wdio/runner: TypeError: Driver.newSession is not a function`

I tracked through all my dependencies and it looks like @wdio/local-runner is the issue and it starts from v8.13.* onwards. 

Then previous versions back to v8.10.0 present a different issue: 

`SyntaxError: Named export ‘sync’ not found. The requested module ‘glob’ is a CommonJS module, which may not support all module.exports as named exports.`

This error refers to: `node:internal/modules/esm/module_job:124:21 and 190:5

All other dependencies can be on latest version but for `@wdio/local-runner` I am stuck with version 8.9.0 for the time being until I can get the time (or suggestions on how) to resolve it. 

Any tips or tricks to debug or address would be appreciated. 

Note: I am in enterprise which prohibits me using GitHub Discussion or copying code to the outside world and I don’t have a personal setup where I could replicate it.

**Comment by christian-bromann** [comment]
It is impossible to reason about this situation without having a reproducible example. I am not sure how the setup looks like and there are a lot of variables here to consider. If you somehow can provide a reproducible example, I can look into the issue.
